### ISSUE TITLE
Forearm pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9906]
**Preferred term label:**
Forearm pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Distal to elbow, radial, ulnar, wrist

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Radial tunnel syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000299
- label: Forearm
- parents: [Pain (HP:0012531)](https://hpo.jax.org/app/browse/term/HP:0012531)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the forearm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9906
- synonyms:

**Comment by MickeySegal** [comment]
Did you mean for the label to be only the one word "forearm"?

**Comment by kanems** [comment]
Can this term be renamed to include "pain" somewhere in the primary concept? I know ontologically it's "pain in the forearm" BUT UMLS is not going to process it based on hierarchy and will likely match it to the CUI for the body part. 
Forearm HP:6000299 to "forearm pain"
Also I think rather than being a direct child of "pain" it would make more sense under  Upper limb pain HP:0012513
